test/onnx/test_operators.py

```diff
@@ -410,9 +410,8 @@ def test_atan(self):
         self.assertONNX(lambda x: x.atan(), x)
 
     def test_flatten(self):
-        # Flatten is a special case of Reshape when the output is a 2-D tensor.
         x = torch.randn(1, 2, 3, 4, requires_grad=True)
-        self.assertONNX(lambda x: x.view(x.size()[0], x.numel() // x.size()[0]), x)
```

Comment:
could you keep the old test case and rename it to something like `test_view_flatten`, and create a new test case for `flattern`